cli/lsp/analysis.rs

```diff
@@ -848,8 +848,26 @@ impl CodeActionCollection {
 
   /// Move out the code actions and return them as a `CodeActionResponse`.
   pub fn get_response(self) -> lsp::CodeActionResponse {
-    self
-      .actions
+    let mut actions = self.actions;
+
+    // Prefer TSC fixes first, then Deno fixes, then Deno lint fixes.
+    actions.sort_by(|a, b| match (a, b) {
+      (CodeActionKind::Deno(a), CodeActionKind::Deno(b)) => {
+        a.title.cmp(&b.title)
+      }
+      (CodeActionKind::DenoLint(a), CodeActionKind::DenoLint(b)) => {
+        a.title.cmp(&b.title)
+      }
+      (CodeActionKind::Tsc(a, _), CodeActionKind::Tsc(b, _)) => {
+        a.title.cmp(&b.title)
```

Comment:
Is this correct? Perhaps we should maintain TypeScript's diagnostic sort order instead of sorting alphabetically?

Reply:
I opened https://github.com/denoland/deno/issues/20544 -- we should restore the sorting order to align with typescript.